lib/std/meta/trait.zig

```diff
@@ -5,11 +5,11 @@ const testing = std.testing;
 
 const meta = @import("../meta.zig");
 
-pub const TraitFn = fn (type) bool;
+pub const TraitFn = fn (type) callconv(.Inline) bool;
 
-pub fn multiTrait(comptime traits: anytype) TraitFn {
+pub inline fn multiTrait(comptime traits: anytype) TraitFn {
```

Comment:
Functions returning `TrainFn` need not be marked `inline`; they're already comptime-only.

Reply:
+